### ISSUE TITLE
Issue on cloud

### ISSUE THREAD

**Issue by alpertrkmn** [s3condAtall/Deezloader#7]
Deezloader worked on localhost(on my computer) and worked when connected with vnc in cloud vps. But it doesnt work when I try connect from my computer (http://serverip:port)  Can anyone run it from vps?

**Comment by s3condAtall** [comment]
@alpertrkmn  Are you using the latest version? The one with deezer api fix?

**Comment by alpertrkmn** [comment]
Yes i am using latest version. It is working on my computer but i cant use it with connection remotely from browser
